### PR TITLE
CORE-20130 - Added logs to help identify the cause of the instability of one of the tests for the Token Selection

### PR DESCRIPTION
**Pull request by filipesoliveira**

### CODE REVIEW

File: components/ledger/ledger-utxo-token-cache/src/main/kotlin/net/corda/ledger/utxo/token/cache/handlers/TokenClaimReleaseEventHandler.kt [hunk 1]
```diff
@@ -23,6 +23,7 @@ class TokenClaimReleaseEventHandler(
         event: ClaimRelease
     ): Record<String, FlowEvent>? {
         log.debug { "Received token claim release for: $event" }
+        log.info( "Received token claim release for claim id ${event.claimId}")
```
Comment (vkolomeyko) [inline components/ledger/ledger-utxo-token-cache/src/main/kotlin/net/corda/ledger/utxo/token/cache/handlers/TokenClaimReleaseEventHandler.kt]: Nit:
```suggestion
        log.info("Received token claim release for claim id ${event.claimId}")
```

File: components/ledger/ledger-utxo-token-cache/src/main/kotlin/net/corda/ledger/utxo/token/cache/handlers/TokenForceClaimReleaseEventHandler.kt [hunk 2]
```diff
@@ -19,11 +19,13 @@ class TokenForceClaimReleaseEventHandler : TokenEventHandler<ForceClaimRelease>
         state: PoolCacheState,
         event: ForceClaimRelease
     ): Record<String, FlowEvent>? {
+        log.info("Force claim release received for claim id ${event.claimId}")
         log.debug { "Received token claim release for: $event" }
```
Comment (vkolomeyko) [inline components/ledger/ledger-utxo-token-cache/src/main/kotlin/net/corda/ledger/utxo/token/cache/handlers/TokenForceClaimReleaseEventHandler.kt:23]:
```suggestion
        log.info("Received token claim release for: $event")
```

File: components/ledger/ledger-utxo-flow/src/main/kotlin/net/corda/ledger/utxo/impl/token/selection/handlers/TokenClaimReleasePostProcessingHandler.kt [hunk 3]
```diff
@@ -22,9 +23,14 @@ class TokenClaimReleasePostProcessingHandler @Activate constructor(
 
     private val serializer = cordaAvroSerializationFactory.createAvroSerializer<Any> {}
 
+    private companion object {
+        private val logger = LoggerFactory.getLogger(this::class.java)
```
Comment (vkolomeyko) [inline components/ledger/ledger-utxo-flow/src/main/kotlin/net/corda/ledger/utxo/impl/token/selection/handlers/TokenClaimReleasePostProcessingHandler.kt:27]:
```suggestion
        val logger = LoggerFactory.getLogger(this::class.java)
```

**Review by vkolomeyko — COMMENTED** [review]
Looks fine - few comments posted

**Comment by filipesoliveira** [comment]
DO NOT MERGE!  FOR TESTING ONLY!

**Pull request closed without merging**